packages/client/src/components/Log.tsx

```diff
@@ -0,0 +1,12 @@
+import { Card, Heading, Text } from 'theme-ui';
+
+export const Log = ({ event }: { event: string }) => {
+  return (
+    <Card sx={{width: "25%"}}>
```

Comment:
you didn't run prettier did you?, in any case I will keep measures, margins etc in the parent

Reply:
I haven't set up `prettier` to run on save on this new machine, just did now and updated, cheers.